content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -155,6 +155,26 @@ Allowed customization are:
 Please note that the configuration field `kubernetesVersion` or the command line flag
 `--kubernetes-version` affect the version of the images.
 
+### Uploading control-plane certificates to the cluster
+
+By adding the flag `--upload-certs` to `kubeadm init` you can temporary upload
+the control-plane certificates to a Secret in the cluster. Please note that this Secret
+will expire automatically after 2 hours. The certificates are encrypted using
+a 32byte key that can be specified using `--certificate-key`. The same key can be used
+to download the certificates when additional control-plane nodes are joining, by passing `--control-plane` and `--certificate-key` to `kubeadm join`.
+
+The re-upload the certificates, the following phase command can be used:
+
+```
+kubeadm init phase upload-certs --upload-certs --certificate-key=SOME_VALUE
```

Comment:
If the key is not provided a new one will be generated; this is simpler for the user (1 command instead of 2)